### DISCUSSION TITLE
Closed PR's showing as open

### DISCUSSION THREAD

**Discussion by isadon** [magit/forge#474] [Support] [answered]
While running forge and doing a fresh pull of all topics on my repo there are various PR's that are showing as open instead of closed. By open I mean that I specified in forge to hide all closed PR's but about 6-7 are showing as open when they are really closed. Trying to visit them does nothing as well.

**Comment by tarsius** [comment, marked as answer]
If you pull on of those topics individually using `n f t`, does that update the state?

Also can you point me to one such pull-request? Does this happen in more than one repository?

**Reply by isadon** [reply]
Doing `N f t` doesn't update the state unfortunately. Is there some other way to get you some info. The PR(s) in question are in a private repo which I can't give access to 🙂.

**Reply by isadon** [reply]
One thing I'm seeing in common with all of them is that the branches the PR's were based on have all since been deleted.

**Reply by tarsius** [reply]
Does `(oref (forge-get-repository t) selective-p) selective-p)` return t? It should not. But that also should not matter if you fetch just one topic.

Have these pull-requests been converted from issues?

Next advice would be to add debug statements to `forge--pull-topic` in `forge-github.el` and maybe related functions.

**Comment by Lenbok** [comment]
I see this as well. I have set the topic limit to not show closed PRs, but the PR list does sometimes include closed ones. In my case as well it is for a PR with merged state, where the branch has since been deleted. The only way I've found to clean up the list is to remove the repository from the forge database and re-add. This is quite annoying.
